lockup/README.md

```diff
@@ -1,43 +1,64 @@
 # Lockup / Vesting contract
 
-# PLEASE WAIT UNTIL [ISSUE 141](https://github.com/near/core-contracts/issues/141) IS RESOLVED BEFORE DEPLOYING THE CONTRACT
-
 ## Overview
 
-This contract acts as an escrow that locks and holds an owner's tokens for a lockup period. A lockup period starts
-from the moment transfers are enabled by voting and lasts for the specified duration.
+This contract acts as an escrow that locks and holds an owner's tokens for a lockup period. 
+The contract contains of lockup and vesting processes that go simultaneously.
```

Comment:
"contains of" is not correct English grammar. You might have meant "consists of"